app/queue.py

```diff
@@ -50,7 +50,7 @@ def inflight_prefix(self, suffix: str = None, process_type: str = None) -> str:
             return f"{Buffer.IN_FLIGHT.value}::{str(process_type)}"
         return f"{Buffer.IN_FLIGHT.value}"
 
-    def inflight_name(self, receipt: UUID = uuid4(), suffix: str = None, process_type: str = None) -> str:
+    def inflight_name(self, suffix: Optional[str], process_type: Optional[str], receipt: UUID = uuid4()) -> str:
```

Comment:
We should keep the order of arguments the same and keep the default values, otherwise the behaviour of the function will change
```suggestion
    def inflight_name(self, receipt: UUID = uuid4(), suffix: Optional[str] = None, process_type: Optional[str] = None) -> str:
```